security.md

```diff
@@ -130,7 +130,7 @@ Add cert paths to your gemspec
 
 -------
 
-### OpenPGP signing is [not recommended due to lack of support](http://www.rubygems-openpgp-ca.org/blog/nobody-cares-about-signed-gems.html).
```

Comment:
How about https://web.archive.org/web/20131125012205/http://www.rubygems-openpgp-ca.org/blog/nobody-cares-about-signed-gems.html

Reply:
Good idea i've amended the commit in 1c4eee3 and added `(archive)` as info on the link